modules/openapi-generator/src/main/resources/python/model_utils.mustache

```diff
@@ -1729,7 +1729,9 @@ def validate_get_composed_info(constant_args, model_args, self):
     var_name_to_model_instances = {}
     for prop_name in model_args:
         if prop_name not in discarded_args:
-            var_name_to_model_instances[prop_name] = [self] + composed_instances
+            var_name_to_model_instances[prop_name] = [self] + list(
+                filter(
+                    lambda x: prop_name in x.openapi_types, composed_instances))
```

Comment:
So technically in every instance of the payload prop_name should be present but this code did work before so I am okay with this being merged back in.